api/controllers/FlairController.js

```diff
@@ -168,8 +168,8 @@ module.exports = {
         }
       });
 
-      var newPFlair = _.get(req, "user.flair.ptrades.current.flair_css_class") || "default";
-      var newsvFlair = _.get(req, "user.flair.svex.current.flair_css_class") || "";
+      var newPFlair = _.get(req, "user.flair.ptrades.flair_css_class", "default");
```

Comment:
Right, but remember that we get rid of `current` when we retrieve the flair in Reddit.js. If we check for it here too, the path is never found, so the user ends up with default flair whenever they set their flair text.